### PR TITLE
Mathematical Appendix.

### PR DESCRIPTION
**Pull request by roconnor-blockstream**

### CODE REVIEW

**Comment by roconnor-blockstream** [comment]
I might rephrase thing later by redefining beta and gamma so that the power sequence start at 1, but I can do that later.  For now I believe the provided appendix contains enough information to now implement error correction.

**Comment by apoelstra** [comment]
ACK. Looks good to me.

File: MasterSeed32.md [hunk 1]
```diff
@@ -439,3 +439,65 @@ This is an example of a [Long MS32 String](#long-ms32-strings).
...
+* `β^79`
+* `β^80`
+* `β^81`
+* `β^82`
+* `β^83`
+* `β^84`
+
+where `β := G ζ`.
+
+`β` is a root of `x^2 + G x + J` and `β` has order 93.
```
Comment (apoelstra) [inline MasterSeed32.md:475]: I'd invert this, like:

```
We extend GF[32] to GF[1024] by adjoining a primitive cube root of unity, `ζ`, satifying `ζ^2 = ζ + P`. We then choose `β := G ζ`, which is a primitive 93th root of unity. Our generating polynomial can then be seen to be the product of (x - β^i) for i in {17, 20, 46, 49, 52 77, 58, 59, 80, 81, 82, 83, 84}.
￼```

File: MasterSeed32.md [hunk 2]
```diff
@@ -439,3 +439,29 @@ This is an example of a [Long MS32 String](#long-ms32-strings).
...
+Below we use the Bech32 character set to denote values in GF[32].
+In Bech32, the letter `Q` denotes zero and the letter `P` denotes one.
+The digits `0` and `2` through `9` do *not* denote their numeric values.
+They are simply elements of GF[32].
+
+The generating polynomial for our BCH code is as follows.
+
+We extend GF[32] to GF[1024] by adjoining a primitive cube root of unity, `ζ`, satifying `ζ^2 = ζ + P`.
+
+We select `β := G ζ` which has order 93, and construct the product `(x - β^i)` for `i` in `{17, 20, 46, 49, 52 77, 78, 79, 80, 81, 82, 83, 84}`.
```
Comment (apoelstra) [inline MasterSeed32.md:456]: I believe this list should be `{17, 20, 46, 49, 52, 59, 77, 78, 80, 81, 82, 83, 84}`. That is, 59 should be there and 79 should not.

Comment (apoelstra) [reply MasterSeed32.md:456]: Wait, lemme double-check that.

Comment (roconnor-blockstream) [reply MasterSeed32.md:456]: 79 should most definitely be there otherwise the code is broken.

Comment (apoelstra) [reply MasterSeed32.md:456]: No, the existing text is correct.

Comment (apoelstra) [reply MasterSeed32.md:456]: Though as a nit, you are missing a comma after `52`

Comment (roconnor-blockstream) [reply MasterSeed32.md]: List is correct, but missing a comma.

**Comment by apoelstra** [comment]
lol ACK.

After spending a quite a lot of time reproducing these values by various exhaustive searches, I agree that they are valuable and not trivial to find, if you don't already have them.

**Pull request merged**